cricPredict/Groups/views.py

```diff
@@ -1,3 +1,19 @@
+from django.http import HttpResponse
 from django.shortcuts import render
+from .forms import GroupCreationForm
+
 
 # Create your views here.
+def create_group(request):
+    form = GroupCreationForm()
+    if request.method == 'POST':
+        form = GroupCreationForm(request.POST)
+        if form.is_valid():
+            instance = form.save(commit=False)
+            instance.admin_id = request.user
+            instance.save()
+            return HttpResponse("Group Created")
+        else:
+            return HttpResponse(form.errors.as_json())
```

Comment:
why are sending error back as JSON, we are not making API calls, errors should be added to the context, and the creation form should ideally be rendered back to the user with error messages.